mne/beamformer/tests/test_lcmv.py

```diff
@@ -139,12 +140,30 @@ def test_lcmv():
         max_stc = stc_max_power.data[idx]
         tmax = stc.times[np.argmax(max_stc)]
 
-        assert_true(0.09 < tmax < 0.11, tmax)
+        assert_true(0.04 < tmax < 0.11, tmax)
```

Comment:
I think the maximum was at 0.65 sec, I will check that again. I followed line 126 here.